.gitignore

```diff
@@ -11,4 +11,16 @@ __pycache__
 
 #IntelliJ
 .idea
-*.iml
\ No newline at end of file
+*.iml
+
+#Setup
+MANIFEST
+build/
```

Comment:
I ended up pulling these out because I wanted GIT to clean them up.  I suggest you do the same.